solana/rpc/data_slice.py

```diff
@@ -0,0 +1,14 @@
+"""DataSlice options.
```

Comment:
I don't think we need an entire module for this since `DataSlice` is only used for two API methods.

Reply:
Moved this as part of the `api` now.